### PR TITLE
CATROID-1155 "Turn the particle effect's additivity off/on" brick

### PR DESCRIPTION
**Pull request by devsebastian**
Ticket: https://jira.catrob.at/browse/CATROID-1155

"Turn the particle effect's additivity off/on" brick

### Your checklist for this pull request
Please review the [contributing guidelines](https://github.com/Catrobat/Catroid/blob/develop/README.md) and [wiki pages](https://github.com/Catrobat/Catroid/wiki/) of this repository.

- [x] Include the name of the Jira ticket in the PR’s title
- [x] Include a summary of the changes plus the relevant context
- [x] Choose the proper base branch (*develop*)
- [x] Confirm that the changes follow the project’s coding guidelines
- [x] Verify that the changes generate no compiler or linter warnings
- [x] Perform a self-review of the changes
- [x] Verify to commit no other files than the intentionally changed ones
- [x] Include reasonable and readable tests verifying the added or changed behavior
- [x] Confirm that new and existing unit tests pass locally
- [x] Check that the commits’ message style matches the [project’s guideline](https://github.com/Catrobat/Catroid/wiki/Commit-Message-Guidelines)
- [x] Stick to the project’s gitflow workflow
- [x] Verify that your changes do not have any conflicts with the base branch
- [ ] After the PR, verify that all CI checks have passed
- [ ] Post a message in the *catroid-stage* or *catroid-ide* [Slack channel](https://catrobat.slack.com) and ask for a code reviewer

### CODE REVIEW

File: catroid/src/main/res/layout/brick_additive_particle_effect.xml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+        <Spinner
+            android:id="@+id/brick_additive_particle_effect_spinner"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:textDirection="locale" />
+
+    </org.catrobat.catroid.ui.BrickLayout>
+
+</LinearLayout>
```
Comment (Rawbat) [inline catroid/src/main/res/layout/brick_additive_particle_effect.xml:56]: Add a newline please :)

**Review by Rawbat — CHANGES_REQUESTED** [review]
FadeParticleEffectAfterWaitTest.catrobat seems to fail at least sometimes on jenkins. If possible please include a fix for that in this PR.

**Review by Rawbat — APPROVED** [review]

File: catroid/src/main/res/layout/brick_additive_particle_effect.xml (hunk 1 above)
Comment (wslany) [reply catroid/src/main/res/layout/brick_additive_particle_effect.xml:56]: Shall I wait for these two things (tests, and/or newline) before merging, or can we do that in another PR?

Comment (Rawbat) [reply catroid/src/main/res/layout/brick_additive_particle_effect.xml:56]: Dev already explained to me that the test can be ingored for now and the missing newline is not a pressing issue. So I would say this PR can be merged.

**Comment by wslany** [comment]
Thanks!

**Pull request merged**